api-mobile/test/activity-with-files.json

```diff
@@ -0,0 +1,54 @@
+{
+  "activityType": "observation",
+  "activitySubType": "plant",
+  "date": "2020-08-30",
+  "locationAndGeometry": {
+    "anchorPointY": 48.3,
+    "anchorPointX": -125.6,
+    "area": 0,
+    "geometry": {
+      "type": "Feature",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -126.6,
+              49.3
+            ],
+            [
+              -125.6,
+              48.3
+            ],
+            [
+              -125.6,
+              49.3
+            ],
+            [
+              -126.6,
+              49.3
+            ]
+          ]
+        ]
+      },
+      "properties": {}
+    }
+  },
+  "activityTypeData": {
+    "pokiness": 17
+  },
+  "activitySubTypeData": {
+    "density": "hard",
+    "colour": "green"
+  },
+  "media": [
+    {
+      "fileName": "bunnyFile",
```

Comment:
Do I even want to know?

Reply:
Haha, I was using a random image of a bunny for testing.

I do notice a small error in this test string though, so Ill fix that